noxfile.py

```diff
@@ -55,7 +62,7 @@
 
 def _get_requirements_filepath(
     category: Literal["docs", "tests", "all"],
-    version: Literal["3.10", "3.11", "3.12", "3.13", "3.14", "3.15"],
+    version: Literal["3.11", "3.12", "3.13"],
```

Comment:
📝 The minimum supported versions have to be repeated here so that the `Literal` construct can be used in _static_ type checking. If we defined it like `Literal[*supported_python_versions*]` then `mypy` wouldn't be able to handle it. 😮‍💨